modules/BookCollectClass.js

```diff
@@ -0,0 +1,20 @@
+export default class BookCollect {
+  constructor() {
+    this.books = [];
+  }
+
+  removeBook(index) {
+    this.books = this.books.filter((item, i) => i !== index);
+    localStorage.setItem('Books', JSON.stringify(this.books));
+    window.location.reload();
+  }
+
+  addBook(BookName, Author) {
+    const book = {
+      name: BookName,
+      author: Author,
+    };
+    this.books.push(book);
+    localStorage.setItem('Books', JSON.stringify(this.books));
+  }
```

Comment:
- [ ] Its a requirement for this milestone to refactor functions to use Arrow Syntax. Kindly do so
![image](https://user-images.githubusercontent.com/81932478/235524800-13e319d4-5b5a-4b48-9f5d-14faba74870e.png)